modules/storages/app/common/storages/peripherals/storage_interaction/nextcloud/set_permissions_command.rb

```diff
@@ -78,24 +87,6 @@ def requested_folder(folder)
       escape_whitespace(folder)
     end
 
-    def error(response)
-      case response
-      when Net::HTTPNotFound
-        error_result(:not_found)
-      when Net::HTTPUnauthorized
-        error_result(:not_authorized)
-      else
-        error_result(:error)
-      end
-    end
-
-    def error_result(code, log_message = nil, data = nil)
-      ServiceResult.failure(
-        result: code, # This is needed to work with the ConnectionManager token refresh mechanism.
-        errors: Storages::StorageError.new(code:, log_message:, data:)
-      )
-    end
```

Comment:
Where did it go?

Reply:
We have the same method, but with different name `error` inside `Errors` module. So, this one is removed in favor for other one from the module.